gspread/auth.py

```diff
@@ -274,14 +288,18 @@ def service_account(filename=DEFAULT_SERVICE_ACCOUNT_FILENAME, scopes=DEFAULT_SC
 
     :param str filename: The path to the service account json file.
     :param list scopes: The scopes used to obtain authorization.
+    :type client_factory: :class:`gspread.ClientFactory`
+    :param client_factory: A factory function that returns a client class.
+        Defaults to :class:`gspread.Client` (but could also use
+        :class:`gspread.BackoffClient` to avoid rate limiting)
 
-    :rtype: :class:`gspread.Client`
+    :rtype: :class:`gspread.ClientFactory`
```

Comment:
here too (basicaly ever occurences of the type ClientFactory in docstring should be updated with `` :class:`gspread.client.Client` ``

Reply:
Changed.